src/dead-file.ts

```diff
@@ -98,7 +98,7 @@ export default function vitePluginDeadFile(
         `All source files: ${sourceFiles.size}`,
         `Used source files: ${touchedFiles.size}`,
         `Unused source files: ${deadFiles.size}`,
-        ...[...deadFiles].map((fullPath) => `  .${fullPath.substring(projectRoot.length)}`),
+        ...[...deadFiles].map((fullPath) => `  /${fullPath.substring(projectRoot.length)}`),
```

Comment:
No this is not an absolute path after remove the projectRoot. The relative path form is  right. And I think the relative form make more sense sine the absolute path is usually redundant with a long prefix.

Reply:
You sure about that ?
Locally with this merge request, and with this config:
```
      deadFile({
        projectRoot: '.',
        include: ['src'],
        exclude: ['src/assets', 'src/styles', 'src/i18n'],
        output: 'dead-files.txt'
      })
```

I've got this kind of output:
`/Users/.../.../.../assets/mouse-click.svg`

Reply:
But I agree, it's redundant and we could just make it relative

Reply:
> You sure about that ? Locally with this merge request, and with this config:
> 
> ```
>       deadFile({
>         projectRoot: '.',
>         include: ['src'],
>         exclude: ['src/assets', 'src/styles', 'src/i18n'],
>         output: 'dead-files.txt'
>       })
> ```
> 
> I've got this kind of output: `/Users/.../.../.../assets/mouse-click.svg`

😯 I think the deadFiles array comes from the result of `readSourceFiles`, which I believe returns an array of absolute path. So where does the `.../.../.../` comes from?

Reply:
Gotcha